src/Symfony/Component/Notifier/Bridge/Nexmo/README.md

```diff
@@ -1,19 +1,7 @@
 Nexmo Notifier
 ==============
 
-Provides [Nexmo](https://nexmo.com) integration for Symfony Notifier.
-
-DSN example
------------
-
-```
-NEXMO_DSN=nexmo://KEY:SECRET@default?from=FROM
-```
-
-where:
- - `KEY` is your Nexmo key
- - `SECRET` is your Nexmo secret
- - `FROM` is your sender
+The Nexmo is deprecated, use the Vonage bridge instead.
```

Comment:
Once it is merged and released, we may need to add a link to the future Vonage Bridge repository here?